mbed_targets/_internal/target_database.py

```diff
@@ -83,9 +83,8 @@ def _response_error_code_to_str(response: requests.Response) -> str:
 def _get_request() -> requests.Response:
     """Make a get request to the API, ensuring the correct headers are set."""
     token = os.getenv(_AUTH_TOKEN_ENV_VAR)
+    header: Optional[Dict[str, str]] = None
```

Comment:
Do you think adding test coverage for the header usage / construction from the env would be beneficial here?

Reply:
Yeah :) I added two tests for the header usage in the last PR. Here I'm only fixing a `mypy` error. The tests are in https://github.com/ARMmbed/mbed-targets/blob/master/tests/test_target_database.py